src/views/More/ShippingReturnPolicy/Page.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import PageHeader from "../../../components/Header/PageHeader";
+import { Expander } from "@components/molecules/Expander";
+import { list } from "./mock";
+import { useState } from "react";
+import { findIndex } from "lodash";
+
+export const Page = (props: any) => {
+  const [listOfItems, showContent] = useState({ value: list });
```

Comment:
I think you just need to store the opened/selected index here.

Reply:
@adrienshen You are right
Done changes.